src/components/leftDrawer.js

```diff
@@ -0,0 +1,301 @@
+  const [forumTag, setForumTag] = React.useState(false);
+  const openForumTag = () => {
+    setForumTag(true);
+  };
+  const closeForumTag = () => {
+    setForumTag(false);
+  };
+
+  const [guideTag, setGuideTag] = React.useState(false);
+  const openGuideTag = () => {
+    setGuideTag(true);
+  };
+  const closeGuideTag = () => {
+    setGuideTag(false);
+  };
+
+  const [tradeTag, setTradeTag] = React.useState(false);
+  const openTradeTag = () => {
+    setTradeTag(true);
+  };
+  const closeTradeTag = () => {
+    setTradeTag(false);
+  };
+
+  const [walletTag, setWalletTag] = React.useState(false);
+  const openWalletTag = () => {
+    setWalletTag(true);
+  };
+  const closeWalletTag = () => {
+    setWalletTag(false);
+  };
+
+  const [settingsTag, setSettingsTag] = React.useState(false);
+  const openSettingsTag = () => {
+    setSettingsTag(true);
+  };
+  const closeSettingsTag = () => {
+    setSettingsTag(false);
+  };
+  
+  if (renderDrawer) {
+    return (
+      <div className="left-drawer">
+        <Drawer
+          variant="permanent"
+          classes={{
+            paper: classes.drawerPaper,
+          }}
+          anchor="left"
+        >
+          <div className={classes.toolbarIcon}>
+            <IconButton onClick={handleDrawerClose}>
+              <ChevronLeftIcon />
+            </IconButton>
+          </div>
+          <Divider />
+          <div classsName="drawer-list-icons">
+            <List className="main-list-icons" autowidth={false}>
+              <MainListIcons
+                location={location}
+                openForumTag={openForumTag}
+                closeForumTag={closeForumTag}
+                openGuideTag={openGuideTag}
+                closeGuideTag={closeGuideTag}
+                openTradeTag={openTradeTag}
+                closeTradeTag={closeTradeTag}
+                openWalletTag={openWalletTag}
+                closeWalletTag={closeWalletTag}
+              />
+            </List>
+            <List className="secondary-list-icons" autowith={false}>
+              <SecondaryListIcons
+                location={location}
+                openSettingsTag={openSettingsTag}
+                closeSettingsTag={closeSettingsTag}
+              />
+            </List>
+          </div>
+        </Drawer>
+        <div className="drawer-list-tags">
+          <div className="main-list-tags">
+            <RenderForumTag forumTag={forumTag} />
+            <RenderGuideTag guideTag={guideTag} />
+            <RenderTradeTag tradeTag={tradeTag} />
+            <RenderWalletTag walletTag={walletTag} />
+          </div>
+          <div className="secondary-list-tags">
+            <RenderSettingsTag settingsTag={settingsTag} />
+          </div>
+        </div>
+      </div>
+    );
+  }
+  return null;
+}
+
+function RenderForumTag({ forumTag }) {
+  if (forumTag) {
+    return (
+      <span className="forum-tag">Forum</span>
+    );
+  }
+  else {
+    return null;
+  }
+}
```

Comment:
Ternary as per above

Reply:
Sorry, see below. First time commenting codes in PR :"

Reply:
Forgot that ternary operator exists, I will fix this on the next commit.